app/src/main/java/com/eveningoutpost/dexdrip/stats/StatsResult.java

```diff
@@ -133,6 +134,27 @@ public double getTotal_insulin() {
         return total_insulin;
     }
 
+    public double getRatio() {
```

Comment:
Could this method be simplified to:

```
 public double getRatio() {
        return getTotal_carbs() / getTotal_insulin();
    }
```

That way if someone's status line contains total carbs or total insulin as well as the ratio then it only has to go to the database once for each component as each method already caches the result.